homeassistant/components/sensor/pollen.py

```diff
@@ -105,7 +105,6 @@
     'maximum': 12
 }]
 
-
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_ZIP_CODE): cv.string,
```

Comment:
For another PR: Would it make sense to validate this as just `str` ? That way it will enforce people to put quotes around their zip code in their config.

Reply:
Ah, that's a great idea! I'll open it up.